src/legacy/api/jlink/__tests__/jlinkFacade-test.js

```diff
@@ -81,45 +51,45 @@ describe('JlinkFacade', () => {
             .then(() => expect(warningMessage).toContain('Lookup failed'));
     });
 
-    it('returns empty map if comName is given, and two serial numbers found in registry, but none are connected', () => {
-        const facade = new JlinkFacade();
-        facade.registry.findJlinkIds = () =>
-            Promise.resolve(['000123456789', '000234567890']);
-        facade.nrfjprogjs.getSerialNumbers = callback =>
-            callback(null, [345678901]);
-        return facade
-            .getSerialNumberMap(['COM1'])
-            .then(map => expect(map.size).toEqual(0));
-    });
+    // it('returns empty map if comName is given, and two serial numbers found in registry, but none are connected', () => {
```

Comment:
Uncomment, remove, or make a comment on when these tests are to be run again

Reply:
Good point. This is part of the `legacy` section of Launcher, which will be removed in its entirety when the last apps (i.e. Trace Collector v1, Link Monitor and Getting Started App) are migrated to the new architecture, so I didn't really bother. But I can add a comment.